infrasim/qemu.py

```diff
@@ -131,10 +152,15 @@ def start_qemu():
     vm.read_from_config()
     cmd = vm.get_qemu_cmd()
     logger.debug(cmd)
-    run_command(cmd, True, None, None)
+    os.system(cmd)
```

Comment:
How to capture the command output? the command output should be very important for debugging why the command failed.